spec/factories/wallets.rb

```diff
@@ -13,6 +13,7 @@
 
     trait :terminated do
       status { 'terminated' }
+      terminated_at { DateTime.now }
```

Comment:
```suggestion
      terminated_at { Time.current }
```

Reply:
Thanks! The rubocop rule only catches `Time.now` but not `DateTime.now`